core/frontend/src/tile/map/MapLayerImageryFormats.ts

```diff
@@ -430,8 +445,43 @@ class ArcGISMapLayerImageryProvider extends MapLayerImageryProvider {
 
   protected get _filterByCartoRange() { return false; }      // Can't trust footprint ranges (USGS Hydro)
   public get maximumZoomLevel() { return this._maxDepthFromLod > 0 ? this._maxDepthFromLod : super.maximumZoomLevel; }
+
+  public uintToString(uintArray: any) {
+    return Buffer.from(uintArray).toJSON();
+
+  }
+
   public async loadTile(row: number, column: number, zoomLevel: number): Promise<ImageSource | undefined> {
-    return super.loadTile(row, column, zoomLevel);
+
+    if (this._settings.status != MapLayerStatus.Valid) {
+      throw new AbandonedError("Layer status not valid.")
+    }
+
+    const tileRequestOptions: RequestOptions = { method: "GET", responseType: "arraybuffer" };
+    tileRequestOptions.auth = this.getRequestAuthorization();
+    try {
+      const tileUrl: string = await this.constructUrl(row, column, zoomLevel);
+      if (tileUrl.length === 0)
+        return undefined;
+
+      const tileResponse: Response = await request(this._requestContext, tileUrl, tileRequestOptions);
+
+      // Check the content type from the response, it might contains an authentication error that need to be reported.
+      if ((tileResponse.header["content-type"] as string).toLowerCase().includes("json")) {

```

Comment:
Checks the tile response for JSON error.  This may change the status of the layer.